pillarbox-analytics/docs/README.md

```diff
@@ -1,95 +1,122 @@
-[![Pillarbox logo](https://github.com/SRGSSR/pillarbox-apple/blob/main/docs/README-images/logo.jpg)](https://github.com/SRGSSR/pillarbox-android)
-[![Last release](https://img.shields.io/github/v/release/SRGSSR/pillarbox-android?label=Release)](https://github.com/SRGSSR/pillarbox-android/releases)
-[![Android min SDK](https://img.shields.io/badge/Android-21%2B-34A853)](https://github.com/SRGSSR/pillarbox-android)
-[![License](https://img.shields.io/github/license/SRGSSR/pillarbox-android?label=License)](https://github.com/SRGSSR/pillarbox-android/blob/main/LICENSE)
+# Module pillarbox-analytics
 
-# Pillarbox Analytics module
+Provides SRG SSR implementation for [Commanders Act](https://www.commandersact.com/) and [ComScore](https://comscore.com/) to send page view events
+and custom events.
 
-Provides SRG SSR implementation for CommandersAct and ComScore to send page view events and custom events.
-
-Custom events are supported only with CommandersAct!
+**Note:** custom events are only supported with Commanders Act.
 
 ## Integration
 
-```gradle
-implementation("ch.srgssr.pillarbox:pillarbox-analytics:$LATEST_RELEASE_VERSION")
+To use this module, add the following dependency to your project's `build.gradle`/`build.gradle.kts` file:
+
+```kotlin
+implementation("ch.srgssr.pillarbox:pillarbox-analytics:<pillarbox_version>")
 ```
 
 ## Getting started
 
-### Configuration and create
+### Configure analytics
 
-Before using `SRGAnalytics` make sure to call `SRGAnalytics.init` first, otherwise it can lead to undefined behavior.
-It is strongly recommended to call the initializer inside your `Application.onCreate` method.
+Before using any functionality, `SRGAnalytics` must be initialized in your [Application][android.app.Application]'s
+[onCreate()][android.app.Application.onCreate] method using either the
+[initSRGAnalytics()][ch.srgssr.pillarbox.analytics.SRGAnalytics.initSRGAnalytics] or the
+[SRGAnalytics.init()][ch.srgssr.pillarbox.analytics.SRGAnalytics.init] method and providing an
+[AnalyticsConfig][ch.srgssr.pillarbox.analytics.AnalyticsConfig] instance.
 
 ```kotlin
-class DemoApplication : Application() {
-
+class MyApplication : Application() {
     override fun onCreate() {
         super.onCreate()
-        
+
         val config = AnalyticsConfig(
             vendor = AnalyticsConfig.Vendor.SRG,
-            nonLocalizedApplicationName = "PillarboxDemo",
```

Comment:
We should keep the `nonLocalizedApplicationName` as it is useful for apps that localized the application name.

`nonLocalizedApplicationName = `You application name not localized here`